tensorflow/python/kernel_tests/cwise_ops_test.py

```diff
@@ -220,8 +222,10 @@ def testFloatBasic(self):
   def testFloatTanhEdge(self):
     x = np.arange(40, 40 + 6).reshape(6).astype(np.float32)
     self._compareBoth(x, np.tanh, tf.tanh)
+    self._compareBothSparse(x, np.tanh, tf.tanh)
     x = np.arange(-40, -40 + 6).reshape(6).astype(np.float32)
     self._compareBoth(x, np.tanh, tf.tanh)
+    self._compareBothSparse(x, np.tanh, tf.tanh, thresh=-40)
```

Comment:
@siddharth-agrawal I see. In that case I don't see any point in this change at all, since the dense and sparse versions call the same underlying tanh function, and there are already tests for the sparse version in the non-edge case.